lib/portlayer/storage/nfs/volume.go

```diff
@@ -107,6 +107,7 @@ func (v *VolumeStore) VolumeCreate(op trace.Operation, ID string, store *url.URL
 	}
 
 	u, _ := v.Service.URL()
+	u.Scheme = "nfs"
```

Comment:
This is a bug in `Service.URL()` then.  Can you make the change there instead of here?  This seems error prone and out of place.